.editorconfig

```diff
@@ -42,6 +42,10 @@ indent_size = 2
 [*.{yml,yaml}]
 indent_size = 2
 
+# Translation files
+[*.xlf]
+indent_size = 2
```

Comment:
Maybe that would be better to assume all files have indent of 2 and make specific case indent 4.